org-code-javabuilder/validation/src/main/java/org/code/validation/TestOutputAdapter.java

```diff
@@ -0,0 +1,11 @@
+package org.code.validation;
+
+import org.code.protocol.ClientMessage;
+import org.code.protocol.OutputAdapter;
+
+public class TestOutputAdapter implements OutputAdapter {
```

Comment:
NIT: Given the java convention of naming the test directory "test" and our convention of using "test" in the names of test files, I'm trying to think if there's a good way to disambiguate this class from the rest of our test code. Perhaps "UserTestOutputAdapter" or something similar?

Reply:
I think we would need a new word to encompass running student unit tests + levelbuilder tests. Currently we are also using "test" for `TestRunner`, which will run both of those tests.

Reply:
thinking more on this, `UserTestOutputAdapter` should work, we would probably also want to update `TestRunner` to `UserTestRunner` and not use "user" when referring just to student unit tests (maybe call them project tests or something?)